src/main/java/org/broadinstitute/hellbender/tools/exome/GetHetCoverage.java

```diff
@@ -98,16 +99,19 @@
     )
     protected int minimumBaseQuality = 20;
 
+    @Argument(
+            doc = "Validation stringency for all BAM files read by this program.  Setting stringency to SILENT " +
+            "can improve performance when processing a BAM file in which variable-length data (read, qualities, tags) " +
+            "do not otherwise need to be decoded.",
+            common=true)
+    protected ValidationStringency VALIDATION_STRINGENCY = ValidationStringency.DEFAULT_STRINGENCY;
```

Comment:
Yeah, the PicardCommandLineProgram-derived tools are a bit behind the
times, though we'll update those at some point as well. This looks much
better now though!

On Tue, Mar 1, 2016 at 8:44 AM, samuelklee <notifications@github.com> wrote:

> In
> src/main/java/org/broadinstitute/hellbender/tools/exome/GetHetCoverage.java
> <https://github.com/broadinstitute/gatk-protected/pull/379#discussion_r54567512>
> :
>
> > @@ -98,16 +99,19 @@
> >      )
> >      protected int minimumBaseQuality = 20;
> >
> > +    @Argument(
> > +            doc = "Validation stringency for all BAM files read by this program.  Setting stringency to SILENT " +
> > +            "can improve performance when processing a BAM file in which variable-length data (read, qualities, tags) " +
> > +            "do not otherwise need to be decoded.",
> > +            common=true)
> > +    protected ValidationStringency VALIDATION_STRINGENCY = ValidationStringency.DEFAULT_STRINGENCY;
>
> thanks for pointing this out. i had actually originally implemented
> something more along the lines of what you suggested below, but then
> changed it after seeing how things were done in
> PicardCommandLineProgram---didn't appreciate that things were being done
> differently for the gatk tools!
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/broadinstitute/gatk-protected/pull/379/files#r54567512>
> .
>



-- 
Chris Norman
Software Engineer/Temp
Broad Institute of MIT and Harvard
email: cnorman@broadinstitute.org <bhill@broadinstitute.org>